CONTRIBUTING.md

```diff
@@ -0,0 +1,5 @@
+# Contributing to PUDL Archiver
```

Comment:
Should this say contributing to PUDL archiver? Or should it be more generic?

Reply:
Good catch! I had started this as a `CONTRIBUTING.md` for the archiver but totally missed this when I copied it over to the org defaults.